<?xml version="1.0"?>
<launch>
  <!-- ====== HSR simulator ===== -->
    <include file="$(find hsrb_gazebo_launch)/launch/hsrb_megaweb2015_world.launch"></include>
    <node pkg="rviz" type="rviz" name="overwite_rviz_conf" args="-d $(find master_project)/rviz_conf/hsr_development.rviz"/>
    <node name="rqt_gui" pkg="rqt_gui" type="rqt_gui"/> 
    <include file="$(find darknet_ros)/launch/darknet_ros.launch">
       <arg name="image" value="/hsrb/head_rgbd_sensor/rgb/image_raw"/>
    </include>

</launch>
